src/components/Main.jsx

```diff
@@ -44,15 +41,6 @@ const Main = ({ handleOpenLoginPopup, isLoggedIn }) => {
       });
```

Comment:
`setLoading(false);` should be in `finally` not to duplicate it in `then` and `catch`